src/bls/bls.h

```diff
@@ -306,14 +306,12 @@ class CBLSPublicKey : public CBLSWrapper<bls::G1Element, BLS_CURVE_PUBKEY_SIZE,
 
 class ConstCBLSPublicKeyVersionWrapper {
 private:
-    bool legacy;
-    bool checkMalleable;
     const CBLSPublicKey& obj;
+    bool legacy;
 public:
-    ConstCBLSPublicKeyVersionWrapper(const CBLSPublicKey& obj, bool legacy, bool checkMalleable = true)
+    ConstCBLSPublicKeyVersionWrapper(const CBLSPublicKey& obj, bool legacy)
             : obj(obj)
```

Comment:
@knst Good catch. But I propose to have it in a separate PR.
(@UdjinM6 will probably agree with me)

Reply:
Agree about keeping bug fixes outside of a more refactoring focused PR